### ISSUE TITLE
plugin could be altering the safe area on iOS 13

### ISSUE THREAD

**Issue by aubrey-fowler** [apache/cordova-plugin-camera#647]
I have an app with Cordova, AngularJS 1.5, and Ionic v1.

```

ionic info

Ionic:

   ionic (Ionic CLI) : 4.10.3 (C:\Users\User\node_modules\ionic)
   Ionic Framework   : ionic1 1.3.5
   @ionic/v1-toolkit : 1.0.22

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : android 8.1.0, browser 5.0.4, ios 5.1.1, windows 4.4.3
   Cordova Plugins       : cordova-plugin-ionic-webview 4.1.3, (and 17 other plugins)

System:

   Android SDK Tools : 26.1.1 (C:\Users\User\Documents\Android\sdk)
   NodeJS            : v10.21.0 (C:\Program Files\nodejs\node.exe)
   npm               : 6.14.4
   OS                : Windows 10
```

I am using this **cordova-plugin-camera** to take photos and access the photo gallery in my app.

config.xml
`    <plugin name="cordova-plugin-camera" spec="^4.0.3" />`

I am using the WkWebView Engine for iOS.

```
    <platform name="ios">
        <preference name="WKWebViewOnly" value="true" />
        <feature name="CDVWKWebViewEngine">
            <param name="ios-package" value="CDVWKWebViewEngine" />
        </feature>
```
css
```
body {
    padding: constant(safe-area-inset-top) constant(safe-area-inset-right) constant(safe-area-inset-bottom) constant(safe-area-inset-left); //iOS 11.2
    padding: env(safe-area-inset-top) env(safe-area-inset-right) env(safe-area-inset-bottom) env(safe-area-inset-left);
}
```
html
`  <meta name="viewport" content="viewport-fit=cover, width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">`

Since then I have been having issues with this plugin.
When I use this plugin, it seems to alter the "safe area" of my application.

Note the safe area and the button is pushed out of it.

![Simulator Screen Shot - iPhone Xr iOS 13 - 2020-08-21 at 14 06 15](https://user-images.githubusercontent.com/1875260/90909570-db950a00-e3cd-11ea-8f6c-114ef16f060f.png)

**Comment by joeldhenry** [comment]
I have the same issue, only just started to notice it! it pushes up my app, so that i cannot see the navigation bar anymore. i tested to find exactly 44px on an iphone X running ios 13.6.1. This is a breaking change for my apps @aubrey-fowler do you know if any older build of the plugin still works as expected? or is it an ios issue?

**Comment by joeldhenry** [comment]
I have the opposite where my screen is being pushed up, rather than being pushed down

**Comment by Pratz007-tech** [comment]
Exactly the same scenario I am going through on ios - 13 . Any solution to it ?

**Comment by PTRFRLL** [comment]
For those of you where the screen is pushed up. Are you sure it's not the statusbar that is disappearing? I had that issue on iOS with the `cordova-plugin-statusbar` plugin.
see here https://github.com/apache/cordova-plugin-statusbar/issues/156

**Comment by jjtfsalgado** [comment]
> For those of you where the screen is pushed up, are you sure it's not the statusbar that is disappearing? I had that issue on iOS with the `cordova-plugin-statusbar` plugin. [apache/cordova-plugin-statusbar#156](https://github.com/apache/cordova-plugin-statusbar/issues/156)

No, we are having that issue without using `cordova-plugin-statusbar`. Only in iOS 13/14
